### PR TITLE
Added TTFB_TESTED_PAGES

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Added documentation for the`TTFB_TESTED_PAGES` environment variable used to configure _Time to First Byte_  page tests to check Magento application performance for sites deployed on Cloud infrastructure.

## Affected DevDocs pages
- https://devdocs.magento.com/guides/v2.3/cloud/env/variables-post-deploy.html
- https://devdocs.magento.com/guides/v2.3/cloud/live/stage-prod-test.html

## Links to Magento source code

[Magento ECE tools pull request 537](https://github.com/magento/ece-tools/pull/537)

### CODE REVIEW

File: guides/v2.1/cloud/env/variables-post-deploy.md [hunk 1]
```diff
@@ -15,14 +15,39 @@ stage:
 
 {% include cloud/customize-build-deploy.md %}
 
+### `TTFB_TESTED_PAGES`
+
+- **Default**— `[]` (an empty array)
+- **Version**—Magento 2.1.4 and later
+
+Configure _Time To First Byte_ (TTFB) testing for specified pages to test your {{ site.data.var.ece }} site performance. For each page to test, specify the path or the HTTP or HTTPS URL.
```
Comment (bbatsche) [inline guides/v2.1/cloud/env/variables-post-deploy.md]: "For each page to page to test, specify the absolute path reference or an absolute URL with protocol and host."

File: guides/v2.1/cloud/live/stage-prod-test.md [hunk 2]
```diff
@@ -209,11 +212,12 @@ We recommend that you review the [Magento Performance Toolkit]({{ site.mage2blob
 
 For best results, we recommend the following tools:
 
-* [Siege](https://www.joedog.org/siege-home/): Traffic shaping and testing software to push your store to the limit. Hit your site with a configurable number of simulated clients. Siege supports basic authentication, cookies, HTTP, HTTPS and FTP protocols.
-* [Jmeter](http://jmeter.apache.org/): Excellent load testing to help gauge performance for spiked traffic, like for flash sales. Create custom tests to run against your site.
+* [Magento application performance test]({{ page.baseurl }}/cloud/env/variables-post-deploy#ttfb_tested_pages)—Test Magento application performance by configuring the TTFB_TESTED_PAGES envrionment variable to test site response time.
```
Comment (bbatsche) [inline guides/v2.1/cloud/live/stage-prod-test.md]: I would put TTFB_TESTED_PAGES in backticks (`)

File: guides/v2.1/cloud/live/stage-prod-test.md [hunk 3]
```diff
@@ -209,11 +212,12 @@ We recommend that you review the [Magento Performance Toolkit]({{ site.mage2blob
...
+* [Siege](https://www.joedog.org/siege-home/)—Traffic shaping and testing software to push your store to the limit. Hit your site with a configurable number of simulated clients. Siege supports basic authentication, cookies, HTTP, HTTPS and FTP protocols.
+* [Jmeter](http://jmeter.apache.org/)—Excellent load testing to help gauge performance for spiked traffic, like for flash sales. Create custom tests to run against your site.
 * New Relic (provided): Helps locate processes and areas of the site causing slow performance with tracked time spent per action like transmitting data, queries, Redis, and so on.
```
Comment (bbatsche) [inline guides/v2.1/cloud/live/stage-prod-test.md]: Should this also have : replaced with a -?

File: guides/v2.1/cloud/live/stage-prod-test.md [hunk 4]
```diff
@@ -209,11 +212,12 @@ We recommend that you review the [Magento Performance Toolkit]({{ site.mage2blob
...
-* [Blackfire]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) (provided): Helps track through the issues New Relic finds and helps you dig deeper into the issue for specifics. Blackfire profiles the environment and helps locate bottlenecks indepth: process, method call, query, load, and so on.
-* [WebPageTest](https://www.webpagetest.org/) and [Pingdom](https://www.pingdom.com/): Real-time analysis of your site pages load time with different origin locations. Pingdom may cost a fee. WebPageTest is a free tool.
+* [Blackfire]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) (provided)x— Helps track through the issues New Relic finds and helps you dig deeper into the issue for specifics. Blackfire profiles the environment and helps locate bottlenecks indepth: process, method call, query, load, and so on.
```
Comment (bbatsche) [inline guides/v2.1/cloud/live/stage-prod-test.md]: Probably did not mean to add that x

File: guides/v2.1/cloud/live/stage-prod-test.md [hunk 5]
```diff
@@ -209,11 +212,12 @@ We recommend that you review the [Magento Performance Toolkit]({{ site.mage2blob
...
```
Comment (meker12) [inline guides/v2.1/cloud/live/stage-prod-test.md]:
```suggestion
* [Blackfire]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) (provided)— Helps track through the issues New Relic finds and helps you dig deeper into the issue for specifics. Blackfire profiles the environment and helps locate bottlenecks indepth: process, method call, query, load, and so on.
```

File: guides/v2.1/cloud/env/variables-post-deploy.md [hunk 6]
```diff
@@ -15,14 +15,39 @@ stage:
 
 {% include cloud/customize-build-deploy.md %}
 
+### `TTFB_TESTED_PAGES`
+
+- **Default**— `[]` (an empty array)
+- **Version**—Magento 2.1.4 and later
+
+Configure _Time To First Byte_ (TTFB) testing for specified pages to test your {{ site.data.var.ece }} site performance. For each page to page to test, specify the absolute path reference or an absolute URL with protocol and host.
```
Comment (meker12) [inline guides/v2.1/cloud/env/variables-post-deploy.md]:
```suggestion
Configure _Time To First Byte_ (TTFB) testing for specified pages to test your {{ site.data.var.ece }} site performance. For each page to test, specify the absolute path reference or an absolute URL with protocol and host.
```

File: guides/v2.1/cloud/live/stage-prod-test.md [hunk 7]
```diff
@@ -209,11 +212,12 @@ We recommend that you review the [Magento Performance Toolkit]({{ site.mage2blob
...
+* [Magento application performance test]({{ page.baseurl }}/cloud/env/variables-post-deploy.html#ttfb_tested_pages)—Test Magento application performance by configuring the `TTFB_TESTED_PAGES` environment variable to test site response time.
+* [Siege](https://www.joedog.org/siege-home/)—Traffic shaping and testing software to push your store to the limit. Hit your site with a configurable number of simulated clients. Siege supports basic authentication, cookies, HTTP, HTTPS and FTP protocols.
+* [Jmeter](http://jmeter.apache.org/)—Excellent load testing to help gauge performance for spiked traffic, like for flash sales. Create custom tests to run against your site.
+* New Relic (provided)—Helps locate processes and areas of the site causing slow performance with tracked time spent per action like transmitting data, queries, Redis, and so on.
+* [Blackfire]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) (provided)— Helps track through the issues New Relic finds and helps you dig deeper into the issue for specifics. Blackfire profiles the environment and helps locate bottlenecks indepth: process, method call, query, load, and so on.
```
Comment (meker12) [inline guides/v2.1/cloud/live/stage-prod-test.md]:
```suggestion
* [Blackfire]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) (provided)—Helps track through the issues New Relic finds and helps you dig deeper into the issue for specifics. Blackfire profiles the environment and helps locate bottlenecks indepth: process, method call, query, load, and so on.
```

**Pull request merged**